Dockerfile

```diff
@@ -8,8 +8,9 @@ SHELL [ "/bin/bash", "-c" ]
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 
-# lock file created by running lock.sh in the top level of the repository
-COPY --chown=mambauser conda-lock.yml /conda-lock.yml
+# explicit specification file file created from pixi.lock by running
+# 'pixi run export' in the top level of the repository
+COPY --chown=mambauser default_linux-64_conda_spec.txt /default_linux-64_conda_spec.txt
```

Comment:
Maybe modify the cmd to create environment directly using pixi?

Reply:
That would definitely be easier, but I assumed that people would be more receptive to an incremental change of having the runtime environment stay with `micromamba` at first. Switching everything to `pixi` solves the error of course too.

Reply:
I'd say go for it, if you like the idea.